config.yml

```diff
@@ -0,0 +1,252 @@
+---
+CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
+docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+
+ci_helper_test: &ci_helper_test ci-helper version -e
+
+ansible:
+  "2.5.5-r0":
+    cmd_test:
+      - python --version
+      - ansible --version
+      - ansible-playbook --version
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PYTHON_VERSION: 2.7.15-r1
+      VERSION: 2.5.5-r0
+
+arachni:
+  "1.5":
+    cmd_test:
+      - arachni --version
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      ARACHNI_VERSION: 1.5.1
+      ARACHNI_WEB_UI_VERSION: 0.5.12
+
+aws:
+  "1":
+    cmd_test:
+      - aws --version
+      - pip list | grep boto3
+      - pip list | grep pyyaml
+      - pip --version
+      - pipenv --version
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PIP_VERSION: 18.0
+      PIPENV_VERSION: 2018.7.1
+      PY_YAML_VERSION: 3.13
+
+chrome:
+  "1":
+    cmd_test:
+      - google-chrome --no-sandbox --version
+      - google-chrome-unstable --no-sandbox --version
+      - node --version
+      - npm --version
+      - yarn --version
+    build_args:
+      MODD_VERSION: 0.5
+
+dind-aws:
+  "1":
+    cmd_test:
+      - aws --version
+      - docker --version
+      - docker-compose --version
+      # - --privileged -d --name dind-aws alpine:latest
+      # - docker ps
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      DOCKER_COMPOSE_VERSION: 1.24.0
+      GLIBC_VERSION: 2.27-r0
+      PIP_VERSION: 18.1
+      PIPENV_VERSION: 2018.10.13
+
+golang:
+  "1.9":
+    cmd_test:
+      - aws --version
+      - go version
+      - glide --version
+      - gin --version
+      - modd --version
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      GLIDE_VERSION: v0.13.1
+      MODD_VERSION: 0.5
+
+java_6_cmd_test: &java_6_cmd_test
+  - java -version
+  - mvn --version
+
+java_8_and_above_test: &java_8_and_above_test
+  - java -version
+  - mvn --version
+  - aws --version
+  - modd --version
+  - *ci_helper_test
+
+java:
+  "6":
+    cmd_test: *java_6_cmd_test
```

Comment:
I don't think we will reuse this test anywhere else, so you can inline it. (and rename `java_8_and_above_test` to `java_test`)

Reply:
Deleted java 10 & update java 6 & java 11 configuration naming.